### PR TITLE
Nfchoa line sources

### PR DESCRIPTION
**Pull request by hagenw**
Hi @narahahn.

I added a few lines to the documentation for your 2.5D and 3D versions of the NFC-HOA driving functions for line sources.

In addition I included some of your remarks, regarding the introduction of spherical coefficients etc.
Its still includes the directional dependency of the source model as I still think that there is some directional dependency. Could you please review my additions and maybe start adding further corrections to the introduction at the beginning as I forgot your proposal we discussed in Rostock.

In order to view the changes on your local machine compile the documentation from the root dir via
`make html-preview` and then go to `_build/html-preview/index.html` and open that file in a browser.

### CODE REVIEW

**Comment by hagenw** [comment]
I updated it to the latest version of our documentation web page and you can have a look at my changes at http://sfstoolbox.org/en/nfchoa_line_sources/.

The functions I added or modified are
http://sfstoolbox.org/en/nfchoa_line_sources/#equation-D_planar
http://sfstoolbox.org/en/nfchoa_line_sources/#equation-line-source-spherical-coefficients
http://sfstoolbox.org/en/nfchoa_line_sources/#equation-D.nfchoa.ls.3D
http://sfstoolbox.org/en/nfchoa_line_sources/#equation-D.nfchoa.ls.2D
http://sfstoolbox.org/en/nfchoa_line_sources/#equation-D.nfchoa.ls.2.5D

File: index.txt [hunk 1]
```diff
@@ -913,7 +913,7 @@ as [Hahn2015]_, eq. (15)
     \breve{S}_n^m(\phi_\text{s},r_\text{s},\w) =
         -\pi \i^{m-n+1}
         \Hankel{2}{m}{\wc r_\text{s}}
-        Y_n^{-m}(\frac{\pi}{2},\phi_\text{s}).
+        Y_n^{-m}(0,\phi_\text{s}).
```
Comment (hagenw) [inline index.txt:916]: \theta = 0 would be inside the horizontal plane, is this the intended direction?

Comment (narahahn) [inline index.txt:916]: Yes, it is supposed to be the plane perpendicular to the line source.

In my paper, \theta indicates the angle from the north pole which is different from here.

**Comment by hagenw** [comment]
OK, thanks, I should read more about line sources ;)

Are you otherwise finished and I can merge it?

**Comment by narahahn** [comment]
It's my great pleasure, Hagen.
I have nothing else regarding the line source driving functions.
I am reading through the entire document and found some points I want to discuss. Should I just write an e-mail or open an issue?

**Comment by hagenw** [comment]
Thanks, I will merge it.

Regarding new discussion, you could decide which way you prefer. If you decide to create an issue, it would be nice to have different issues for different parts of the discussion. But it is also fine to start with one mail and then create pull requests with solutions for the different points.

**Pull request merged**